### DISCUSSION TITLE
Multiple advertising account management

### DISCUSSION THREAD

**Discussion by Ss110525** [amzn/ads-advanced-tools-docs#245] [Q&A] [answered]
I have more than 10 accounts in US
I'd like to bring every data regarding Ads just as Campaign Reports, Search Term Reports, etc.
As I understand Amazon document to connect API, the following step is below

if i as direct advertiser apply Join Amazon Ad Partner Network, is it  okey? 
Will it cause store association?

**Comment by elizafarley** [comment, marked as answer]
Hi @Ss110525, if you apply as a direct advertiser, you can still associate more than one advertising account with your client ID.

See these definitions to determine whether you fit into the partner or direct developer category:

- Partner : Businesses that build applications and software solutions to automate and help optimize advertising on behalf of others can access the Amazon Ads API as a third party.
- Direct Advertiser : Advertisers that use the Amazon Ads API to automate, scale, and optimize advertising activities and reporting.
